distributed/utils.py

```diff
@@ -618,3 +618,18 @@ def shutting_down(l=_shutting_down):
     to early bind this function rather than look it up when calling it,
     since at shutdown module globals may be cleared.
     """
+
+
+def open_port(host=''):
+    """ Return a probably-open port
+
+    There is a chance that this port will be taken by the operating system soon
```

Comment:
Bokeh supports this, it's getting the information back that's a problem.  perhaps that isn't necessary though.